components/buttongroup/metadata/buttongroup.yml

```diff
@@ -1,14 +1,19 @@
 name: Button group
 SpectrumSiteSlug: https://spectrum.adobe.com/page/button/
 sections:
+  - name: Migration Guide
+    description: |
+      ### 1/24/2024 - Version 7.0.0
```

Comment:
@pfulton 
Curious your thoughts are adding the date and version number here? Also, am I understanding correctly that as a breaking change the version number would bump from 6.2.0 to 7.0.0?

Reply:
I love this! I'm not convinced that the date is needed, but including the version is important for future-proofing this and being able to determine what may or may not be relevant in the docs.

Reply:
Yeah, this is nice! Thank you for adding it!